### PR TITLE
adding some genetic techniques

### PR DESCRIPTION
**Pull request by lzehl**
related to #180

### CODE REVIEW

**Comment by lzehl** [comment]
@MaaikevS or @UlrikeS91 could one of you review this PR?

**Comment by UlrikeS91** [comment]
> @MaaikevS or @UlrikeS91 could one of you review this PR?

Sure. I'll do it now.

File: instances/technique/pathwayAnalysis.jsonld [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/pathwayAnalysis",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "A 'pathway analysis' refers to a group of techniques that aim discover what biological themes, and which biomolecules, are crucial to understand biological pathways of (typically) high-throughput biological data (adapted from [García-Campos et al., 2015](https://doi.org/10.3389/fphys.2015.00383)).",
```
Comment (UlrikeS91) [inline instances/technique/pathwayAnalysis.jsonld]: "... a group of techniques that aim TO discover ..."

File: instances/technique/wholeGenomeSequencing.jsonld [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/wholeGenomeSequencing",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "A 'whole genome sequencing' is a genetic test (sequencing technique) to determine the entire, or nearly the entire, DNA sequence of an organism's genome at a single time. [adapted from [wikipedia](https://en.wikipedia.org/wiki/Whole_genome_sequencing)]",
```
Comment (UlrikeS91) [inline instances/technique/wholeGenomeSequencing.jsonld]: Remove the "a" in the beginning. Just " 'Whole genome sequencing' is a ..."

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/technique/pathwayAnalysis.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/pathwayAnalysis.jsonld]: done

File: instances/technique/wholeGenomeSequencing.jsonld (hunk 2 above)
Comment (lzehl) [reply instances/technique/wholeGenomeSequencing.jsonld]: done

**Review by UlrikeS91 — APPROVED** [review]

**Pull request merged**